### ISSUE TITLE
[Feature]: Could you please offer .deb for version 2.4.0-rc.1

### ISSUE THREAD

**Issue by wangyiran33** [milvus-io/milvus#31750]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

I'm very excited and grateful for your support for the multi-vector retrieve and sparse vector. Could you please provide the deb file for version 2.4.0-rc.1 because this is the only installation method I can use in my environment?

**Comment by xiaofan-luan** [comment]
/assign @LoveEachDay 
Do we have deb file in any of our previous release?

**Comment by wangyiran33** [comment]
> /assign @LoveEachDay Do we have deb file in any of our previous release?

Yes, you have provided deb files for every release except this one. Looking forward to getting the new version of the deb file.😊

**Comment by LoveEachDay** [comment]
@wangyiran33 We only provide deb package for stable release? Could you use milvus docker image instead for 2.4.0-rc.1?

**Comment by wangyiran33** [comment]
> @wangyiran33 We only provide deb package for stable release? Could you use milvus docker image instead for 2.4.0-rc.1?

Okay, thank you anyway.
